letsdrill.rb

```diff
@@ -1,12 +1,29 @@
 def get_letter_grade(integer)
  
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
+  when 80...90
+    return "B"
+  when 70...80
+    return "C"
+  when 60...70
+    return "D"
+  else
+    return "F"
+  end
+  
 
 end
 
 def shortest_string(array)
 
   #Put your code here!
+  sorted_array = array.sort_by(&:length)
```

Comment:
Correct me if I'm wrong, but the `&` converts the object to a block.  The object is the length method, denoted by the symbol `:length`, which is called on every item that `sort_by` yields